tensorflow-framework/src/test/java/org/tensorflow/framework/optimizers/GradientDescentTest.java

```diff
@@ -22,7 +22,6 @@
 import org.tensorflow.op.Ops;
 import org.tensorflow.op.core.Assign;
 import org.tensorflow.op.core.Constant;
-import org.tensorflow.op.core.Init;
```

Comment:
The disabled determinism test uses `Init`, so you should fix the reference there too.

Reply:
Yeah I haven't updated `framework` yet.